web/libs/datamanager/src/components/Common/Userpic/Userpic.scss

```
@@ -0,0 +1,67 @@
+.userpic {
+  width: 28px;
+  height: 28px;
+  display: flex;
+  font-size: 12px;
+  position: relative;
+  align-items: center;
+  border-radius: 50%;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.08);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
  background: rgba(var(--black), 0.08);
```